elbepack/initvmaction.py

```diff
@@ -148,6 +153,50 @@ def initvm_state(self):
         return self.initvm.info()[0]
 
 
+def is_qemu_running():
```

Comment:
`def is_soap_port_reachable()` would be more clear.
The function is not specific for qemu.